### ISSUE TITLE
Questions about classification model in MultiMNIST

### ISSUE THREAD

**Issue by MarcuXu** [yobibyte/unitary-scalarization-dmtl#4]
Hey, I noticed that you utilized a modified encoder-decoder version of the LeNet in C.1.1 MultiMNIST in your paper. However, upon examining the code, I observed that only 2 fully connected layers `self._fc1, self._fc2 = nn.Linear(50, 50), nn.Linear(50, 10)` are employed in "MultiLeNetO". I'm curious about how its test accuracy can reach 95% despite the absence of non-linear layers such as conv. And where are the "encoder-decoder"?
